plugins/modules/keycloak_client_rolemapping.py

```diff
@@ -63,6 +63,33 @@
             - Name of the group to be mapped.
             - This parameter is required (can be replaced by gid for less API call).
 
+    parents:
```

Comment:
This is a copy of a documentation from https://github.com/ansible-collections/community.general/blob/779a5d99f5500bf3b228783b44c4cc32c1c6fbd2/plugins/modules/keycloak_group.py#L79 slightly adjusted